Presentation.Web/Controllers/API/V1/OrganizationRegistrationsController.cs

```diff
@@ -16,14 +16,13 @@
 namespace Presentation.Web.Controllers.API.V1
 {
     [PublicApi]
-    [RoutePrefix("api/v1/organization-registrations")]
-
-    public class OrganizationRegistrationController: BaseApiController
+    [RoutePrefix("api/v1/organization-units")]
+    public class OrganizationRegistrationsController: BaseApiController
     {
         private readonly IOrganizationUnitService _organizationUnitService;
         private readonly IOrgUnitService _orgUnitService;
 
-        public OrganizationRegistrationController(IOrganizationUnitService organizationUnitService,
+        public OrganizationRegistrationsController(IOrganizationUnitService organizationUnitService,
```

Comment:
I don't know how to name this controller, it performs "OrganizationUnit" actions, but there is already an OrganizationUnitController (that controller uses the GenericHierarchyApiController)